vulnz/src/main/java/io/github/jeremylong/vulnz/cli/commands/CveCommand.java

```diff
@@ -72,6 +72,7 @@ public class CveCommand extends AbstractNvdCommand {
      * Start year (until today) to cache CVEs for.
      */
     private static final int START_YEAR = 2002;
+    private static final int EARLIEST_CVE_YEAR = 1988;
 
```

Comment:
We should either name the variables in a way it is obvious or document why both exist, or the next time, it will be not implemented the right way again :)